[DEVX-4943] Terminus 4 documentation and release note. (#9546)
pantheon-systems/documentation: source/content/terminus/02-install.md

</Accordion>

**Terminus requires the following:**

> - PHP Version 8.2 or later

Reviewer: @jazzsequence is going to make edits to this page to make the compatibility combinations clearer

Reviewer 2: I think this is probably no longer necessary.

<Alert title="PHP 8.4 and Terminus" type="danger">

> Terminus, 3.x, is not compatible with PHP 8.4. PHP <= 8.3 should be used with Terminus 3.x.

Reviewer 2: ```suggestion
Terminus, 3.x, is not compatible with PHP 8.4. If you are using PHP <= 8.3 you should use Terminus 3.x.
```

> Terminus, 3.x, is not compatible with PHP 8.4. If you are using PHP <= 8.3 you should use Terminus 3.x.

Reviewer 3: ```suggestion
Terminus 3.x is not compatible with PHP 8.4. If you are using PHP <= 8.3 you should use Terminus 3.x.
```

@kporras07

> Terminus, 3.x, is not compatible with PHP 8.4. If you are using PHP <= 8.3 you should use Terminus 3.x.

Reviewer 4: This should say `PHP <= 8.1`, as Terminus 4 works with PHP 8.2+.  We should probably also recommend folks upgrade if they're in this situation.

> Terminus 4.x is compatible with PHP 8.4 and can be installed now.

Reviewer 4: PHP 8.2+

Reviewer 2: ```suggestion
Terminus 4.x is compatible with PHP 8.2+.
```

- Make the file executable
- Add a symlink to your local `bin` directory for the Terminus executable

<TabList>

> <Tab title="PHP 8.4+" id="php84" active={true}>

Reviewer 4: 8.2+

Reviewer 2: This is confusing. We're breaking it down by when to install Terminus 3 vs Terminus 4. The reason you'd want to install Terminus 4 is because you are using PHP 8.4. I guess I should break that down in the titles instead.

```bash{promptUser: user}
mkdir -p ~/terminus && cd ~/terminus
curl -L https://github.com/pantheon-systems/terminus/releases/download/4.0.0/terminus.phar --output terminus
chmod +x terminus
./terminus self:update
sudo ln -s ~/terminus/terminus /usr/local/bin/terminus
```

</Tab>

> <Tab title="PHP 8.2/8.3" id="php82-83">

Author: This looks confusing; maybe wrong titles? 8.2 and 8.3 are also supported by Terminus 4 (and it would be recommended). You should only go to 3.6.2 if you need 7.4-8.1

Reviewer 2: Fair

```suggestion
<Tab title="PHP 8.3 and below" id="php83-">
```

> <Tab title="PHP 8.3 and below" id="php83-">

Reviewer 4: 8.1 and below

```bash{promptUser: user}
mkdir -p ~/terminus && cd ~/terminus

> curl -L https://github.com/pantheon-systems/terminus/releases/download/4.0.0/terminus.phar --output terminus

Reviewer: @jazzsequence to add a subheading for PHP 8.2/8.3 instructions
